### DISCUSSION TITLE
How do i clear a textbox using CTK

### DISCUSSION THREAD

**Discussion by Ezed-On** [TomSchimansky/CustomTkinter#1160] [General]
So i've been trying to clear a textbox contents and it wont work i need help can someone please provide me help, Thank you

**Comment by legopitstop** [comment]
To clear the contents of a Textbox use the [delete](https://github.com/TomSchimansky/CustomTkinter/wiki/CTkTextbox#deleteself-index1-index2none) method.
```py
text = CTkTextbox()
text.pack()
text.delete(0.0, 'end')
```
